CHANGELOG.md

```diff
@@ -61,6 +61,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [Datumaro] Added `convert` command to convert datasets directly (<https://github.com/opencv/cvat/pull/1837>)
 - [Datumaro] Added an option to specify image extension when exporting datasets (<https://github.com/opencv/cvat/pull/1799>)
 - [Datumaro] Added image copying when exporting datasets, if possible (<https://github.com/opencv/cvat/pull/1799>)
+- Source type support for tags, shapes and tracks (<https://github.com/opencv/cvat/pull/1192>)
```

Comment:
Wrong place. Could you please move these lines into the section for next release?

Reply:
Updated